src/auth/index.js

```diff
@@ -13,9 +13,17 @@ const EMBEDDED_USER_AUTH_DATA_PARTS = 4;
 
 export default class Auth {
   constructor(client, rootUrl, options) {
+    let namespace;
+    if (client === null || client.clientAppId === '') {
+      namespace = '<admin>';
```

Comment:
[nit] For safety purposes, what do you think of prefixing the client namespace with something like `client-${client.clientAppId}`? That way we can never mistake admin for client.

Reply:
Maybe `client.${client.clientAppId}`. Full stop might be easier to differentiate. I have no strong opinion either way.

Reply:
sure